app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarValidatorTest.kt

```diff
@@ -12,23 +15,40 @@ class BloodSugarValidatorTest {
   @Test
   fun `when blood sugar reading is empty, return error`() {
     val bloodSugarReading = ""
-    val result = bloodSugarValidator.validate(bloodSugarReading)
+    val result = bloodSugarValidator.validate(bloodSugarReading, Random)
 
     assertThat(result).isEqualTo(ErrorBloodSugarEmpty)
   }
 
   @Test
   fun `when blood sugar reading is more than maximum possible, return error`() {
     val bloodSugarReading = "1001"
-    val result = bloodSugarValidator.validate(bloodSugarReading)
+    val result = bloodSugarValidator.validate(bloodSugarReading, PostPrandial)
 
     assertThat(result).isEqualTo(ErrorBloodSugarTooHigh)
   }
 
+  @Test
+  fun `when hba1c blood sugar reading is more than maximum possible, return error`() {
```

Comment:
I didn't want to include the value in the test name, because it felt it's an implementation detail. If the maximum/minimum value is increased in the future we have to update the test names to match those values.

Reply:
The min and max values are in fact not an implementation detail. They are very much part of the spec itself. These values have been defined as a feature requirement so they should be included in the test as well.